bin/compile

```diff
@@ -18,7 +35,17 @@ cd $BUILD_DIR
 if [ -f shard.yml ]; then
   echo "-----> Installing Dependencies"
   $CRYSTAL_DIR/bin/crystal deps
+
+  if [ -f app.cr ]; then
```

Comment:
Let's keep this for now, but with a message that says something like `DEPRECATED: Your main file should have the name of your shard and not app.cr\nCompiling app.cr` .

What do you think?